Hardware/cheshire.md

```diff
@@ -0,0 +1,134 @@
+---
+riscv_hardware: true
+cmake_plat: cheshire
+xcompiler_arg: -DRISCV64=1
+platform: Cheshire
+arch: RV64IMAFDC
+virtualization: "No"
+iommu: "No"
+simulation_target: false
+Status: "Unverified"
+Contrib: "UNSW"
+Maintained: "UNSW"
+cpu: Cheshire
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2024 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Cheshire
+
+Cheshire is a 6-stage RISC-V CPU. For details, refer to
+[https://github.com/pulp-platform/cheshire](https://github.com/pulp-platform/cheshire)
+
+Cheshire currently only provides seL4 support for the [Genesys 2
+board](https://reference.digilentinc.com/reference/programmable-logic/genesys-2/reference-manual)/
+
+## Building the GCC toolchain
+
+{% include risc-v.md %}
+
+## Building seL4test
+
+Running seL4test requires a checkout of the Cheshire OpenSBI fork. This can be performed as follows
+from the root directory of seL4test after initialising the repo.
+   ```
+   cd tools/opensbi
+   git remote add pulp https://github.com/pulp-platform/opensbi
+   git fetch pulp && git checkout pulp/cheshire
+   ```
```

Comment:
Do we need this now that we don't package OpenSBI with the sel4test image for Cheshire?

Reply:
No, it's not needed with the most recent changes. I've removed this.